spec/models/post_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe Post, type: :model do
+  describe "Associations" do
+    it "should belong to user" do
+      post = build(:post)
```

Comment:
With rspec, we usually try to keep setup outside the `it` block using some combinatin of `subject`, `let`, `let!`, and `before`. This is a decent but not amazing post about the different things you can do: https://medium.com/devnetwork/step-by-step-guide-to-write-rspec-that-is-understandable-and-readable-30279b04dd43. All you need to do here though is move post up into a `subject` block.

Reply:
Just want to make sure this looks ok in my new commit before I merge